nknd.go

```diff
@@ -375,3 +384,112 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func GetChordID(seeds []string, publickey []byte) ([]byte, error) {
+	id, err := chain.DefaultLedger.Store.GetID(publickey)
+	if err == nil {
```

Comment:
We should probably log error here because even if node join for the first time, the expected err should still be nil